web-server/v0.4/.circleci/config.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
```

Comment:
What does this file get used for?

Reply:
This file is used for testing but we will be implementing a different gray box testing framework in future. This will be removed under this PR.